secant/Util/SeedIndex.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  SeedIndex.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 17.02.2022.
+//
+
+@propertyWrapper
+struct SeedIndex {
+    var wrappedValue: Int {
+        didSet { wrappedValue = min(24, max(1, wrappedValue)) }
```

Comment:
How is this different to what the Clamped property wrapper does? (except for the range that is now hardcoded).

Reply:
The SeedIndex in only syntactical and let's say contextual change. Instead of having a universal range clamp property that can be set to anything, hardcoding it as a SeedIndex would clarify the use case more. 

Let's stick with the Clamped for now but I modified it for a good reason.

here's the new code
```swift
@propertyWrapper
struct Clamped<Value: Comparable> {
    var value: Value
    let range: ClosedRange<Value>
    var wrappedValue: Value {
        get {
            min(
                max(range.lowerBound, value),
                range.upperBound
            )
        }
        set { value = newValue }
    }

    init(wrappedValue: Value, _ range: ClosedRange<Value>) {
        self.value = wrappedValue
        self.range = range
    }
}
```

Notice that I swapped bounding function from set to get. The reason is that at initialisation time the set is not called so the clamped index would hold the initial value even out of bounds as long as newValue is set.

The previous version of Clamped was used in 
```swift
struct EnumeratedChip: View {
    @Clamped(1...24)
    var index: Int = 1
```
so anytime EnumeratedChip is initialised 
```swift
// example
var myView = EnumeratedChip(index: 101, text: "dummy")
print(myView.index) // prints 101
```
The reason is `init` is called, `get` is called with the simple { value } set to 101.